.gitignore

```diff
@@ -1,8 +1,9 @@
-*~
+﻿*~
 .DS_Store
 _atc-call-stack.js
 _site/
 coverage/
+.vs/
```

Comment:
Is this for Visual Studio?

Reply:
Yes. I'll remove it if you want, I was just getting *very* annoyed with it. (I use Atom now, pretty happy but the linter isn't wonderful)

Reply:
you can leave it, I was just curious.

Reply:
you could try VSCode?  I've moved on from Atom to VSCode recently and am really liking it.  It's also quite a bit faster than Atom.